src/components/unit/ImportUnitsPage.tsx

```diff
@@ -0,0 +1,54 @@
+import { Typography } from '@material-ui/core';
+import Container from '@material-ui/core/Container';
+import React from 'react';
+import { UnitMergeReview } from 'units/UnitMergeReview';
+
+import { SelectImportFile } from 'components/common/SelectImportFile';
+import { UnitsImportWithValidation } from 'generated/sdk';
+import { StyledPaper } from 'styles/StyledComponents';
+import useDataApiWithFeedback from 'utils/useDataApiWithFeedback';
+
+export const getFileContents = async (file: File): Promise<string> => {
+  return new Promise((resolve) => {
+    const reader = new FileReader();
+    reader.readAsText(file);
+    reader.onload = () => {
+      resolve(reader.result as string);
+    };
+  });
+};
+
+export default function ImportUnitsPage() {
+  const { api } = useDataApiWithFeedback();
+  const [validationResult, setValidationResult] =
+    React.useState<UnitsImportWithValidation | null>(null);
+
+  return (
+    <Container>
```

Comment:
I see that we usually use `StyledContainer` at the pages like this but maybe here you have some reason not to use it. This is because of alignment of paddings and margins with other views. Maybe here it is already aligned like this but just wanted to be sure.